dts/bindings/i3c/i3c-controller.yaml

```diff
@@ -0,0 +1,78 @@
+# Copyright (c) 2022 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+
+# Fields for I3C Controllers
+
+include: base.yaml
+
+bus: i3c
+
+properties:
+    "#address-cells":
+      required: true
+      const: 3
+      description: |
+        For I3C devices, the 3 fields are static address, first half
+        of Provisioned ID, and the second half of the Provisioned ID.
+          1. The static address can be assigned to be the target device's
+             dynamic address before Dynamic Address Assignment (DAA)
+             starts. Can be zero if static address is not used, and
+             target address is determined via DAA.
+          2. First half of the Provisioned ID contains the manufacturer
+             ID left-shifted by 1, where the manufacturer ID is
+             the bits 33-47 (zero-based) of the 48-bit Provisioned ID.
+          3. Second half of the Provisioned ID contains the combination of
+             the part ID (bits 16-31 of the Provisioned ID) left-shifted
+             by 16, and the instance ID (bits 12-15 of the Provisioned ID)
+             left-shifted by 12. Basically, this is the lower 32 bits
+             of the Provisioned ID.
+
+        Note that the DT node of a I3C target device should be in format
+        "<node name>@<static address>", e.g. "sensor@42".
+
+        For I2C devices, the 3 fields are static address, 0x00, and
```

Comment:
This relates to the address cell which is about the children. This information is duplicated in `i3c-device.yaml`. I can remove it, but thought it might be a good idea to have it here also.

Reply:
ah, I see... but it's probably best if it's just within the `i3c-device.yaml`